src/rules/max-expects.ts

```diff
@@ -0,0 +1,89 @@
+import { AST_NODE_TYPES } from '@typescript-eslint/utils';
+import {
+  FunctionExpression,
+  createRule,
+  isExpectCall,
+  isTypeOfJestFnCall,
+} from './utils';
+
+export default createRule({
+  name: __filename,
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description: 'Enforces a maximum assertion calls in a test body',
```

Comment:
```suggestion
      description: 'Enforces a maximum number assertion calls in a test body',
```

Reply:
Fixed in https://github.com/jest-community/eslint-plugin-jest/pull/1166/commits/d96cca7a682d639b072ea3097912178f90cbb610